README.md

```diff
@@ -59,9 +59,10 @@ import (
 )
 
 func RequireFedora() error {
-	// The OSRelease instance
-	or, err := osrelease.New()
-
+	// The OSRelease instance using the default paths
+	or, err := osrelease.New(nil)
+	// Or to inspect the files in /sysroot
+	//or, err := osrelease.New(osrelease.PrefixDefaultFilePaths("/sysroot"))
```

Comment:
Probably just a matter of taste, but I was think of leaving `New` as-is and having an additional constructor like `osrelease.WithOverrides(sysroot string, usrMountpoint string)`.

Reply:
That works for me. I'm fine with reworking this to make a secondary constructor. @Rahuls0720 WDYT?

Reply:
I agree with @lucab

Reply:
Updated